{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "1167a1d4_83a02c99",
        "filename": "lib/pel_utils.py",
        "patchSetId": 8
      },
      "lineNbr": 136,
      "author": {
        "id": 1000027
      },
      "writtenOn": "2022-12-12T10:08:15Z",
      "side": 1,
      "message": "oops seems that there is no - try and exception block for this function.. please add it.\n\nAnd align the lines to fit in 80 character limit.",
      "revId": "ced0b673380f7561c3f6eba9fc57577a0b50d3bd",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "66621213_ba301e60",
        "filename": "lib/pel_utils.py",
        "patchSetId": 8
      },
      "lineNbr": 154,
      "author": {
        "id": 1000027
      },
      "writtenOn": "2022-12-12T10:08:15Z",
      "side": 1,
      "message": "event ID",
      "range": {
        "startLine": 154,
        "startChar": 19,
        "endLine": 154,
        "endChar": 24
      },
      "revId": "ced0b673380f7561c3f6eba9fc57577a0b50d3bd",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}